_posts/2022-12-29-vanilla-os-22-10-kinetic.md

```diff
@@ -44,7 +44,7 @@ conflicting packages.
 
 It achieves this by installing software inside one or more containers fully
 managed by Apx having restricted access to your system’s resources while still
-being able to use the same drivers, display server, etc.
+being able to use the same drivers, displays, etc.
```

Comment:
```suggestion
being able to use the same drivers, hardware, graphical apps, etc.
```
This is the one used for distrobox. I seem to have missed this when writing the docs.

Reply:
You can't run host GUI applications from inside the container, can you?

Reply:
> You can't run host GUI applications from inside the container, can you?

Yeah, you can't in distrobox they are meaning Wayland/X11. But I guess we can leave the hardware alone.

Reply:
You mean both drivers and hardware? That sounds good to me.

Reply:
> You mean both drivers and hardware? That sounds good to me. 

Sure I guess.